docs/source/kedro_project_setup/settings.md

```diff
@@ -31,6 +31,8 @@ Every Kedro project comes with a default pre-populated `pyproject.toml` file in
 package_name = "package_name"
 project_name = "project_name"
 kedro_init_version = "kedro_version"
+tools = ""
```

Comment:
If we are including tools here we should probably include `example_pipeline` as well